src/tests/components/forecastSummary.test.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummary from "../../components/ForecastSummary";
+
+describe("ForecastSummary", () => {
+  const validProps = {
+    date: 1525046400000,
+    description: "Stub description",
+    icon: "800",
+    temperature: {
+      min: 12,
+      max: 11,
+    },
+    onSelect: () => {},
+  };
+
+  it("renders correctly", () => {
+    const { asFragment } = render(
+      <ForecastSummary
+        date={validProps.date}
+        description={validProps.description}
+        icon={validProps.icon}
+        temperature={validProps.temperature}
+        onSelect={validProps.onSelect}
+      />
+    );
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("it renders correct values for props", () => {
+    const { getByText, getByTestId } = render(
+      <ForecastSummary
+        date={validProps.date}
+        description={validProps.description}
+        icon={validProps.icon}
+        temperature={validProps.temperature}
```

Comment:
Theres a warning in the console when running these tests caused by not included all the required props. If youre not sure about this you can always look in the component you are testing to validate what exactly should be getting passed in, in this case the `onSelect` prop needs adding in:
```
onSelect={validProps.onSelect}
```